tests/test_dcos_e2e/backends/docker/test_docker.py

```diff
@@ -377,7 +382,80 @@ class TestNetworks:
     Tests for Docker container networks.
     """
 
-    def test_default(self):
+    @pytest.fixture()
+    def docker_network(self) -> Iterator[Network]:
+        """
+        Return a Docker network.
+        """
+        client = docker.from_env(version='auto')
+        ipam_pool = docker.types.IPAMPool(
+            subnet='172.28.0.0/16',
+            iprange='172.28.0.0/24',
+            gateway='172.28.0.254',
+        )
+        network = client.networks.create(
+            name='dcos-e2e-network-{random}'.format(random=uuid.uuid4()),
```

Comment:
Comment that this name starts with `dcos-e2e` because that matches the network name prefix in the `Makefile`.